cis_env.py

```diff
@@ -0,0 +1,115 @@
+import cis_config as conf
+import random
+import numpy as np
+import dna_decoding
+from cis_cell import random_vector_of_length
+
+
+class Curl():
+    def __init__(self, coor_origin, vec_len):
+        self.coor_origin = coor_origin
+        self.vec_len = vec_len
+
+    def _P(self, x, y, z):
+        nx = 0
+        return nx
+
+    def _Q(self, x, y, z):
+        ny = -z + self.coor_origin[1]
+        return ny
+
+    def _R(self, x, y, z):
+        nz = y - self.coor_origin[2]
+        return nz
+
+    def get(self, x, y, z):
+        old_pos = np.array([x, y, z])
+        curl_pos = np.array([
+            self._P(x, y, z),
+            self._Q(x, y, z),
+            self._R(x, y, z),
+        ])
+
+        return curl_pos * self.vec_len
+
+
+curl = Curl(
+    np.array([
+        conf.WORLD_DIMENSION[0] / 2,
+        conf.WORLD_DIMENSION[1] / 2,
+        conf.WORLD_DIMENSION[2] / 2
+    ]),
+    0.002
+)
+
+
+def feed_cell(cell, cell_dict, already_averaged_dict, food_factor=1):
+    f = random.uniform(0, 1)
+    if f < dna_decoding.food_theshold(cell.dna):
```

Comment:
Doesn't the food threshold decode the probability for a cell to find food. Why should it be part of the cells DNA?

Reply:
It shouldn't be 😅 
I just used that to make sure this works as expected

Reply:
I would suggest to swap that out as soon as we have the food function and proper interactions between cells that are not connected